crates/opencascade-sys/build.rs

```diff
@@ -1,35 +1,45 @@
-use occt_sys::{occt_include_path, occt_lib_path};
+const OCCT_VERSION: (u8, u8) = (7, 2);
```

Comment:
Nit: it would be nice to document that this is a semver requirement.

Reply:
This might need to be moved into the relevant `detect()` function to avoid an unused const warning from clippy.